crates/bevy_ui/src/layout/mod.rs

```diff
@@ -214,7 +212,30 @@ without UI components as a child of an entity with UI components, results may be
         };
         for root_nodes in camera_root_nodes {
             self.taffy
-                .compute_layout(root_nodes.implicit_viewport_node, available_space)
+                .compute_layout_with_measure(
+                    root_nodes.implicit_viewport_node,
+                    available_space,
+                    |known_dimensions: taffy::geometry::Size<Option<f32>>,
+                     available_space: taffy::geometry::Size<taffy::AvailableSpace>,
+                     _node_id: taffy::tree::NodeId,
+                     node_context: Option<&mut Box<dyn Measure>>| {
+                        let size = node_context
+                            .map(|measure| {
+                                measure.measure(
+                                    known_dimensions.width,
+                                    known_dimensions.height,
+                                    available_space.width,
+                                    available_space.height,
+                                )
+                            })
+                            .unwrap_or(Vec2::ZERO);
```

Comment:
Should layouting an empty leaf node with no size log a warning/error?

Reply:
I'd like a `warn_once!` for this.

Reply:
I'd say probably not. Such a node could get it's size from it's parent (e.g. by stretch alignment in a flexbox stack or to a grid cell)